### PR TITLE
Update Welcome Intro page on docs #133

### PR DESCRIPTION
**Pull request by hnjolles1**
See:   https://github.com/livepeer/docs/issues/133

### CODE REVIEW

File: docs/installation/install-livepeer/installation-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- Gas fees L2 -- Ethereum to cover gas fees - (Also ***See*** LPT2 migration information )
+
+#### Bandwidth
+- Internet Bandwidth: each stream requires about 6Mbs up/down per stream
+e.g., 1660 super 20 streams -- 120 Mbs/sec for that video card.
+[Livepeer bandwidth requirements](/video-miners/reference/bandwidth)
+[Test your bandwidth](/video-miners/reference/bandwidth#test-your-bandwidth)
+[Calculating your bandwith](https://www.calculator.net/bandwidth-calculator.html) 
+- Minimum and Ideal Internet Bandwidth -- 120 mbs
+- - Network Knowledge -- Understanding static and dynamic IP addresses, firewalls is necessary as you change configurations during set-up and troubleshooting
```
Comment (hthillman) [inline docs/installation/install-livepeer/installation-tutorial.md:51]: does this belong under the bandwidth header?

File: docs/installation/install-livepeer/installation-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+### LPT -- Livepeer Token
+
+- Required LPT - required [Livepeer token](/protocol/core-concepts/token.md) for staking for those who perform work on the network
+- Livepeer explorer
+- Optimization for increased access
+
+There are 100 available slots. In order to see where you can fit in, go to the livepeer explorer to see how 
+- Gas fees L2 -- Arbitrum to cover gas fees - (Also ***See*** LPT2 migration information )
```
Comment (hthillman) [inline docs/installation/install-livepeer/installation-tutorial.md:64]: looks like some of this is duplicated from above?

File: docs/installation/livepeer-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+1. Check the specified requirements and pre-requisites  
+2. Follow the Installation workflow for 
+> - [video miners](/livepeer-quick-start/#start-video-mining)
+> - [developers](/livepeer-quick-start/#start-developing)
+
+>**Recommendations** If you are new to Livepeer, you can familiarize yourself futher about our platform capabilities highlighted in [Core Concepts](/core-concepts/core-concepts.md). 
+
+### Technical Requirements
+
+At Livepeer we support:
```
Comment (hthillman) [inline docs/installation/livepeer-quickstart.md:17]: how about:
"`go-livepeer` supports:"?

two reasons - first, we want to be clear that go-livepeer is just one potential implementation; second, I want to move away from implying that the company controls everything

File: docs/installation/livepeer-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+8. Vote
+9. Troubleshooting
+
+
+You will be able to:
+1. Configure your broadcasting preferences
+2. Integrate with a CDN
+3. Withdraw your broadcaster funds
+4. Enable verification (experimental)
+5. Troubleshoot Live Peer
```
Comment (hthillman) [inline docs/installation/livepeer-quickstart.md:85]: `Livepeer`

File: docs/intro.md [hunk 5]
```diff
@@ -1,26 +1,68 @@
...
-new topics, add new how-to guides, and provide examples wherever you think it
-might be helpful. All documentation is editable via
-[GitHub](https://github.com/livepeer/livepeer-org/tree/master/docs).
+This site is organized by topics including concepts, tasks, and reference information designed for [Orchestrators](/livepeer-quick-start/video-mining-quick-start/choosing-a-role.md) on the Livepeer Network including:
 
-## Something missing? {#something-missing}
+- [Video Miners](/video-miners/core-concepts/roles-and-responsibilities#types-of-video-miners), including [Transcoders](/video-miners/core-concepts/roles-and-responsibilities#types-of-video-miners)
+- [Video Developers](/video-developers/overview)
+
+**Note** [Delegators](/protocol/core-concepts/ecosystem-participants#delegator) can also benefit from content on this site, with some information included for you as well.
```
Comment (hthillman) [inline docs/intro.md:16]: it might just be a stylistic preference but this sentence reads a little odd to me.

File: docs/intro.md [hunk 6]
```diff
@@ -1,26 +1,68 @@
...
 
 -   Edit this article using the link at the bottom of the page. If this is your first time editing a page, please check out the [contribution guide](/contributing/overview).
 -  [File an issue](https://github.com/livepeer/docs) for us. To avoid duplicates, we ask that you do a quick search of existing issues to see if any similar issues exist.
--   Come talk to us in [Discord](https://discord.gg/uaPhtyrWsF)
\ No newline at end of file
+-   Come talk to us in [Discord](https://discord.gg/uaPhtyrWsF)
+
+
+### What's new
+
```
Comment (hthillman) [inline docs/intro.md:25]: yesss, love this section

Comment (hthillman) [reply docs/intro.md:25]: wow, fantastic

File: docs/video-miners/overview.md [hunk 7]
```diff
@@ -8,6 +8,23 @@ sidebar_position: 1
 This section of the livepeer.org documentation is for **video miners**. Video
 miners in the Livepeer network provide video transcoding services.
 
+Orchestrators are nodes running on the Livepeer network
+bitrates that need to be transcoded in realtime on the network
```
Comment (hthillman) [inline docs/video-miners/overview.md:12]: perhaps this wasn't meant to be in this PR?

File: docs/installation/install-livepeer/installation-tutorial.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+sidebar_position: 1
+title: Installation Tutorial
+---
+
+This tutorial is for those new to video mining with a brief overview of how Livepeer works, with contextual information about the steps to setup a Livepeer Orchestrator to earn passive income utilizing your current GPUs for video mining on the Livepeer network.
+
+## About video mining on Livepeer
+
+### Livepeer Explorer 
+
```
[the remark this answers is not in the document]
Comment (hnjolles1) [reply docs/installation/install-livepeer/installation-tutorial.md:11]: I'm working on this now, this shouldn't be here.

**Pull request closed without merging**